onprem-environment

```diff
@@ -8,3 +8,5 @@ DB_PASS=admin
 DB_NAME=installer
 SERVICE_BASE_URL=http://127.0.0.1:8090
 DEPLOY_TARGET=onprem
+DUMMY_IGNITION=false
+OPENSHIFT_INSTALL_RELEASE_IMAGE=quay.io/ocpmetal/ocp-release:v1.0.9.1
```

Comment:
Both changes above need to be added to config/onprem-iso-config.ign and config/onprem-iso-fcc.yaml.

Reply:
I added the release image and left the ignition file at version 3.0.0.

DUMMY_IGNITION defaults to `false`, and I only put it here so it's easy for devs to turn it on.